spec/javascripts/actions/projectBoard_spec.js

```diff
@@ -1,30 +1,84 @@
-import { expandStoryIfNeeded } from '../../../app/assets/javascripts/actions/projectBoard';
-import { toggleStory } from '../../../app/assets/javascripts/actions/story';
+import { 
+  expandStoryIfNeeded, closeSearch, 
+  closeSearchSuccess, sendErrorNotificationIfNeeded
+} from '../../../app/assets/javascripts/actions/projectBoard';
+import { toggleStory, receiveStories } from '../../../app/assets/javascripts/actions/story';
+import { sendErrorNotification } from '../../../app/assets/javascripts/actions/notifications';
 
 describe('Project Board Actions', () => {
-  it('Should dispatch toggleStory when storyId is true', async () => {
-    const storyId = 127;
+  describe('expandStoryIfNeeded', () => {
+    it('Should dispatch toggleStory when storyId is true', async () => {
```

Comment:
I believe you could refactor theses `it`s for something like:
```
expandStoryIfNeeded
  - When storyId is true
    - dispatches toggleStory
  - When storyId is false
    - does not dispatch toggleStory
```
and follow the same pattern with `sendErrorNotificationIfNeeded` and `closeSearch` tests 😄